src/components/dev/LocalizationReport.js

```diff
@@ -7,21 +7,31 @@ import {
   List,
   Modal,
 } from 'semantic-ui-react';
+
 import _ from 'lodash';
 import { getKeyPathsArray, getKeyPathStrings } from '../../utils/objectKeyPaths';
 import { getLocalizationData } from '../../localization/all';
 
+
 // Get copy of localization data
 const localizations = getLocalizationData();
 
 const ReportItem = function ({ keyPath, test, locKey, pass }) {
+
+  let iconName = `check square`,
+      color    = `green`;
+  if (!pass) {
+    iconName = `window close`;
+    color    = `red`;
+  }
+
   return (
     <List.Item>
       <List.Icon
-        name={ pass ? 'check square' : 'window close' }
-        size='large'
-        color={ pass ? 'green' : 'red' }
-        verticalAlign='middle' />
+        name          = { iconName }
+        color         = { color }
+        size          = { `large` }
```

Comment:
Question for discussion: do we want to change string prop values to be `propName="propValue"` instead of ```propName={ `propValue` }```? It's the convention I see most often, and makes JSX look a little more like HTML.

Reply:
Definitely @todo for a discussion. Feel free to add the issue if you want.

Reply:
*future discussion